src/d/actor/d_a_pirate_flag.cpp

```diff
@@ -23,7 +23,8 @@ void daPirate_Flag_packet_c::setNrmVtx(cXyz*, int, int) {
 
 /* 0000083C-00000E44       .text draw__22daPirate_Flag_packet_cFv */
 BOOL daPirate_Flag_packet_c::draw() {
-    /* Nonmatching */
```

Comment:
it's best to leave the Nonmatching comment in until the function is fully matched so it's clear at a glance it needs more work